### PR TITLE
Add readme to projects

### PR DESCRIPTION
**Pull request by samleeflang**
Add readme.
https://naturalis.atlassian.net/browse/DD-233

### CODE REVIEW

**Review by southeo — DISMISSED** [review]
So many commas :') looks good

File: README.md [hunk 1]
```diff
@@ -1,22 +1,130 @@
...
+As we use generic objects, the API documentation is not as detailed as we would like.
+We are looking at additional options for more detailed documentation.
+
+### Digital Specimens
+For Digital Specimen, we only provide search and read functionality through the APIs.
+It follows a generic structure.
+`{endpoint}/{prefix}/{suffix}`
+Optionally, this is followed by a version `/{version}` for a specific version.
+It can also be followed by a particular view on the data such as `/full`
+The full endpoint provide all specimen information including all connected information.
```
Comment (TomDijkema) [inline README.md:50]: provide's' -> add s at the end of provide

**Review by TomDijkema — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,22 +1,130 @@
...
 
 # Backend DiSSCo
-The backend provides the backend API's for the frontend.
-It handles the logic and preprocesses the items for frontend optimization.
-It consists of a set of REST API's which return JSON. 
-Find/Search API's will be open, API's for create/update/delete will be protected by Oauth (Keycloak).
-Data is retrieved from the `dissco` database and elasticSearch.
+The DiSSCo backend provides APIs which can be used by other applications, such as [DiSSCover](https://sandbox.dissco.tech/).
+It handles the logic and preprocesses the items for serialization.
+In general the find/search APIs are open and APIs for create/update/delete are protected by Oauth (Keycloak).
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply README.md:6]: :+1:

File: README.md [hunk 3]
```diff
@@ -1,22 +1,130 @@
...
 
-## API's
-The API's are document with OpenAPI v3 and a Swagger endpoint.
+All endpoints are based on the [JSON:API](https://jsonapi.org/) specification.
+It follows the guidelines and best practices described in [BiCIKL Deliverable 1.3](https://docs.google.com/document/d/1RgngKSPabEs-Pir6vA25iFDgVorbEZe7duT7L7vQ7QI)
+The organisation endpoints form an exception on this.
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply README.md:10]: :+1:

File: README.md [hunk 4]
```diff
@@ -1,22 +1,130 @@
...
+It can also be followed by a particular view on the data such as `/full`
+The full endpoint provide all specimen information including all connected information.
+This means all annotations, all connected digital media objects and all annotations on the connected digital media objects.
+We also provide a JSON-LD view on the data on the `/jsonld` endpoint.
+This endpoint does not comply the JSON:API standard, but follows the [JSON-LD](https://json-ld.org/) implementation.
+
+Additionally, there are several aggregation and search endpoints.
+In general, they can by filtered by using the following structure.
+`/search?country=Netherlands&midsLevel=1&country=Finland`
+Multiple key=value pairs can be used.
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply README.md:58]: :+1:

File: README.md (hunk 1 above)
Comment (samleeflang) [reply README.md:50]: :+1:

**Review by TomDijkema — APPROVED** [review]

**Pull request merged**